docs/source/adb.rst

```diff
@@ -0,0 +1,131 @@
+
+  a. To check to see if you have java installed, enter the following command into the Terminal::
+
+        java -version
+
+  b. If JDK version 8 is not installed, install it with the following command:
+
+    1. On Ubuntu version 14.04::
+
+        sudo add-apt-repository ppa:webupd8team/java
+        sudo apt-get update
+        sudo apt-get install oracle-java8-installer
+
+    2. On Ubuntu 16.04::
+
+        sudo apt install default-jre
+
+2. Open your internet browser and go to `the Android developer website <https://developer.android.com/studio/index.html#Other>`_ .
+3. Scroll down to *Get just the command line tools*. Download the SDK tools package.
+4. Unzip the file into your chosen directory.
+5. In the downloaded Linux SDK tools, start the Android SDK Manager by executing the program **android** in *android-sdk-linux/tools* like this::
+
+        cd YOUR_ANDROID_SDK_LOCATION/android-sdk-linux/tools
+        ./android
+
+6. Perform the following steps in the Android SDK Manager.
+
+  a. Deselect everything except for *Android SDK Platform - tools*.
+  b. Click **Install** once finished.
+  c. Android Debug Bridge (adb) should now be installed to *YOUR_ANDROID_SDK_LOCATION/android-sdk-linux/platform-tools*.
+
+7. Add adb to your PATH.
+
+  a. Edit your `~/.bashrc` file and add this line::
+
+        export PATH=${PATH}:YOUR_ANDROID_SDK_LOCATION/android-sdk-linux/platform-tools
+
+  b. Save `.bashrc` and then call::
+
+        source .bashrc
+
+  c. Confirm that adb is in your PATH by calling the following command::
+
+        which YOUR_ANDROID_SDK_LOCATION/android-sdk-linux/platform-tools/adb
+
+  d. The result of this command should be::
+
+        adb: YOUR_ANDROID_SDK_LOCATION/android-sdk-linux/platform-tools/adb
+
+8. Continue to :ref:`final-install` below to complete installation.
+
+
+.. _final-install:
+
+----------------------------------
+Final Installation (All Platforms)
+----------------------------------
+
+1. Enable USB Debugging on your phone.
+
+  a. On Android devices:
+
+    1. Tap seven (7) times on the Build Number listed under *Settings -> About Phone*.
+    2. Then, under *Settings -> Developer Options*, enable USB debugging.
+
+  b. On Amazon Kindle Fire:
+
+    1. Tap seven (7) times on the Serial Number listed under *Settings -> Device Options*.
+    2. Then, under *Settings -> Device Options -> Developer Options*, turn on Enable ADB.
+
+2. Connect your Android device to your computer via USB. When the *Allow USB Debugging?* popup displays, tap **OK**.
+3. Add adb to your PATH environment variable.
+
+  a. Right-click the Start menu and select System.
+  b. Select *Advanced System Settings -> Advanced -> Environment Variables*.
+  c. Under *User Variables*, select *PATH* and click **Edit**.
+  d. Under *Edit Environment Variables*, click **New** and add the path to adb (e.g. C:\Program Files (x86)\Android\android-sdk\platform-tools).
```

Comment:
double backticks for paths

I also wonder whether it would be easier to read if we wrapped menu selection in backticks too to make it stand out from the surrounding text .. `Advanced System Settings -> Advanced -> Environment Variables`